rancher/v1.4/en/kubernetes/ingress/index.md

```diff
@@ -464,3 +464,52 @@ spec:
 ```
 
 In our configuration, the `defaults` and `global` keywords identify the customizable sections and should be followed by a new line. Every parameter in these sections should be followed by a new line.
+
+
+#### Example of a Global Load Balancer
+
+You can set up a ingress that is launched using as many load balancers as there are hosts, i.e. each host has one load balancer. 
+
+Example `global-ingress.yml`
+
+```yaml
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: test
+  annotations:
+    config: "defaults\n balance source\nglobal\nmaxconnrate 60"
+    io.rancher.scheduler.global: "true"
```

Comment:
We should add a comment in the yml in case people just want to read the yml. 

# This annotation allows the ability to create load balancers on every host